### ISSUE TITLE
Not a real Problem, only a question

### ISSUE THREAD

**Issue by Heikoho** [OctoPrint/OctoPrint-FirmwareUpdater#123]
Hi,

Firmware Update ist running ok.
I have only one little problem, but i get myself a Workaround.
before updating i must send the Gcode M22. Then it works perfect.
My only  question is:
i can only send a Command (not a Gcode) BEFORE starting the update.
Is there a possibility to send the Gcode M22 over the Pre Command-section???
what must the command look like?

Thank you very much.

Greetings
Heiko

**Comment by benlye** [comment]
Duplicate of #113
